server/server_test.go

```diff
@@ -42,12 +40,14 @@ func TestRunReservedPort(t *testing.T) {
 		nil,
 	)
 
-	if _, ok := err.(*net.OpError); !ok {
-		t.Fatalf("Received unexpected err: %s", err.Error())
-	}
-	if !strings.Contains(err.Error(), "bind: permission denied") {
-		t.Fatalf("Received unexpected err: %s", err.Error())
-	}
+	assert.Error(t, err)
+	assert.IsType(t, &net.OpError{}, err)
+	assert.True(
```

Comment:
Non-blocking - assert also has an `assert.Contains` which calls `string.Contains` for you.  This is great though, comment was just in case you wanted to use it in the future.